CMakeLists.txt

```diff
@@ -639,6 +640,28 @@ if (BUILD_TESTING)
         endforeach(benchmark)
 
     endif()
+
+    if (S2N_INTEG_TESTS)
+        find_package (Python3 COMPONENTS Interpreter Development)
+        file(GLOB integv2_test_files "${PROJECT_SOURCE_DIR}/tests/integrationv2/test_*.py")
+        foreach(test_file_path ${integv2_test_files})
+            get_filename_component(test_filename ${test_file_path} NAME_WE)
+            string(REGEX REPLACE "^test_" "" test_name ${test_filename})
+            string(CONCAT test_target "integ_" ${test_name})
+
+            add_custom_target(${test_target}
```

Comment:
Would it make sense to instead use the `add_test` directive and then we can run them with normal CTest commands?

https://github.com/aws/s2n-tls/blob/a924960293dc4da8c059cf0cc9a9f345cbaa3a6d/CMakeLists.txt#L589

Reply:
We could add them to the over all list of tests with add tests. We would need to exclude them from the list so its possible to just run the unit tests by themselves. We could then use ctest to run them individually. 

Here is an example of how that could be done: https://stackoverflow.com/questions/61572034/ctest-disable-a-set-of-labeled-tests-by-default-but-run-them-when-explicitly. 

Is this preferable?

Reply:
I think it's generally a good idea to put tests there so there's a unified interface for running tests. Then you get all of the test selection features that ctest provides for free.